apps/daimo-clippy/package.json

```diff
@@ -3,8 +3,9 @@
   "version": "1.0.0",
   "main": "src/index.ts",
   "scripts": {
-    "dev": "echo Use 'npm start' for Clippy dev",
+    "dev": "ts-node-dev src/index.ts",
     "start": "tsx src/index.ts",
+    "start:v2": "ts-node src/index.ts",
```

Comment:
ah, i see. what was the tsx error?

Reply:
Weird, just rebuilt and it's no longer giving an error. Wonder if it might have been b/c I had copied node_modules over at first. Undid these changes though.

Reply:
Weird, just rebuilt and it's no longer giving an error. It probably is b/c I had copied node_modules over at first. Undid the changes 👍